components/SStatus.vue

```diff
@@ -29,12 +63,63 @@
     },
 
     computed: {
-      _items() {
-        return [ ...this.items ].map(item => ({
-          ...item,
-          name: truncate(item.name, this.truncateLength)
-        }));
+      _verified() {
+        return [...this.items]
+          .filter(item => item.verified)
+          .map(item => itemWithName(item, this.truncateLength))
+          .sort((itemA, itemB) => itemA.blockchainEntry.timestamp - itemB.blockchainEntry.timestamp);
+      },
+
+      _notVerified() {
+        return [...this.items]
+          .filter(item => !item.verified)
+          .map(item => itemWithName(item, this.truncateLength));
       }
     }
   }
 </script>
+
+<style lang="scss" scoped>
+  @import '~assets/scss/breakpoint.scss';
+
+  .v-card.verified {
+    display: flex;
+    flex-direction: row;
```

Comment:
Any reason for divining the `flex-direction` to `row` since `row` is the default?

Reply:
Nope. Fixed.